cross/libmysqlclient/Makefile

```diff
@@ -0,0 +1,55 @@
+PKG_NAME = libmysqlclient
+PKG_REAL_NAME = mysql-connector-c
+PKG_VERS = 6.1.6
```

Comment:
Is there a reason why cross/libmysqlclient and native/libmysqlclient does not compile same version? If none, I expect here 6.1.11

Reply:
Thanks, this is a mistake of mine.

Reply:
@hgy59 May you please fix version so that I may publish